lib/create-contentful-api.ts

```diff
@@ -234,8 +234,7 @@ interface BaseClient {
    * const response = await client.getTags()
    * console.log(response.items)
    */
-  // TODO type query
-  getTags(query?: any): Promise<TagCollection>
+  getTags(query?: TagQueries): Promise<TagCollection>
```

Comment:
We need some tests to check whether the tag query has the correct format.

Reply:
Sure I will add the tests